ccd-definition/CaseEvent/MultiState.json

```diff
@@ -150,7 +150,7 @@
     "Name": "Upload a C2",
     "Description": "Upload a c2 to the case",
     "DisplayOrder": 4,
-    "PreConditionState(s)": "Submitted;CLOSED",
+    "PreConditionState(s)": "Submitted;Gatekeeping;PREPARE_FOR_HEARING;CLOSED",
```

Comment:
what about access control for uploadC2-PREPARE_FOR_HEARING, it is still CRU for cafcass and solicitor